recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 
 # Start+Enable the lvmetad service on RHEL7, it is required by default
 # but not automatically started
-if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7
+if node['platform_family'] == 'rhel' && node[platform] != 'amazon' && node['platform_version'].to_i >= 7
```

Comment:
This is a fresh `amazon linux 2017` spinned up from scratch (no post configurations)

```
[ec2-user@ip-172-31-20-128 ~]$ sudo service lvm2-lvmetad status
lvmetad (pid  1891) is running...
[ec2-user@ip-172-31-20-128 ~]$ cat /etc/os-release 
NAME="Amazon Linux AMI"
VERSION="2017.03"
ID="amzn"
ID_LIKE="rhel fedora"
VERSION_ID="2017.03"
PRETTY_NAME="Amazon Linux AMI 2017.03"
ANSI_COLOR="0;33"
CPE_NAME="cpe:/o:amazon:linux:2017.03:ga"
HOME_URL="http://aws.amazon.com/amazon-linux-ami/"
```

i think it's got `lvm2-lvmetad` service by default, just not using `systemd`
```
[ec2-user@ip-172-31-20-128 ~]$ sudo systemctl status lvm2-lvmetad
sudo: systemctl: command not found
```